protocol/xmrmaker/swap_state.go

```diff
@@ -410,29 +410,37 @@ func (s *swapState) exit() error {
 	}
 }
 
-func (s *swapState) reclaimMonero(skA *mcrypto.PrivateSpendKey) (mcrypto.Address, error) {
+func (s *swapState) reclaimMonero(skA *mcrypto.PrivateSpendKey) error {
+	// write counterparty swap privkey to file in case something goes wrong
```

Comment:
I would say "to disk" or "to the database", otherwise it sounds like an independent file.